testsuite/run_sets/reposync.yml

```diff
@@ -14,5 +14,6 @@
 - features/reposync/srv_enable_sync_products.feature
 - features/reposync/srv_wait_for_reposync.feature
 - features/reposync/srv_check_reposync.feature
+- features/reposync/srv_add_rocky8_repositories.feature
```

Comment:
not verified but don't you need to remove a srv_add_rocky7_repositories.feature ?

ignore if braindead please